Change the Shopping Cart Sidebar field to the Mini Cart field (#1192)
magento/merchdocs: src/configuration/sales/checkout.md

|--- |--- |--- |
|Display Cart Summary|Website|Determines the value that appears in parenthesis after the My Cart link. Options: Display number of items in cart / Display item quantities|

## Mini Cart

![]({% link images/images/config-sales-mini-cart.png %}){: .zoom}
[*Mini Cart*]({% link sales/mini-cart.md %})

|Field|[Scope]({% link configuration/scope.md %})|Description|
|--- |--- |--- |

> |Display Mini Cart|Store View|Determines if a mini-cart appears in the shopping cart. The display of the mini cart depends upon the theme. Options: Yes / No|

Reviewer: ```suggestion
|Display Mini Cart|Store View|Determines if the mini cart appears on store pages when the cart icon in the header is clicked. The display of the mini cart depends upon the theme. Options: Yes / No|
```

> |Number of Items to Display Scrollbar|Store View|Determines the number of items that can appear in the mini cart before the scrollbar is triggered.|

Reviewer: ```suggestion
|Number of Items to Display Scrollbar|Store View|Determines the number of items that can appear in the mini cart before the scrollbar is triggered. Default: 5|
```

> |Maximum Number of Items to Display|Store View|Determines the maximum number of items that can appear in the mini cart.|

Reviewer: ```suggestion
|Maximum Number of Items to Display|Store View|Determines the maximum number of items that can appear in the mini cart. Default: 10|
```
